docs/site/sidebars/lb4_sidebar.yml

```
@@ -242,73 +269,83 @@ children:
       url: Database-migrations.html
       output: 'web, pdf'
 
-  - title: 'Using TypeORM'
-    output: 'web, pdf'
-    children:
-    - title: 'How to use TypeORM with LoopBack'
-      url: Using-typeorm-with-loopback.html
+    - title: 'Using TypeORM'
       output: 'web, pdf'
+      children:
+      - title: 'How to use TypeORM with LoopBack'
+        url: Using-typeorm-with-loopback.html
+        output: 'web, pdf'
 
-  - title: 'Running and debugging apps'
-    url: Running-and-debugging-apps.html
+  - title: 'Accessing Services'
     output: 'web, pdf'
     children:
+      - title: 'Call Other Services'
+        url: Calling-other-APIs-and-web-services.html
+        output: 'web, pdf'
 
-    - title: 'Setting debug strings'
-      url: Setting-debug-strings.html
-      output: 'web, pdf'
+  - title: 'Validation'
```

For consistency, let's keep using `-ing` form in the names of how-to guide sections please.

```suggestion
  - title: 'Validating data'
```

It would be great to use the same style for individual pages too, but considering the amount of work required, I think it's better to leave this second part for later.